src/components/EventCalender/Yearly/YearlyEventCalender.spec.tsx

```
@@ -313,18 +375,82 @@
-    const { container } = renderWithRouter(
-      <Calendar
-        eventData={multiMonthEvents}
-        refetchEvents={mockRefetchEvents}
-      />,
+    // Ensure all router mocks are properly set up for this test
+    vi.mocked(useParams).mockReturnValue({ orgId: 'org1' });
+
+    // Use ErrorBoundary to catch potential errors
+    const ErrorFallback = ({ error }: { error: Error }) => (
+      <div data-testid="error-message">{error.message}</div>
     );
```

ESLint

[error] Declare only one React component per file

(react/no-multi-comp)